README.md

```diff
@@ -582,6 +582,18 @@ The API Supports a number of options to configure the read
      </td>
      <td>Read</td>
    </tr>
+   <tr valign="top">
```

Comment:
Please split to another PR, otherwise the documentation will be available before the release

Reply:
Removed this. WIll open a new PR after release